bin/midea-ac

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env ruby
+
+require 'thor'
+require 'yaml'
+require 'midea_air_condition'
+
+class MideaCLI < Thor
+  CONFIG_PATH = File.expand_path('~/.midea/config')
+
+  desc 'configure', 'Configure client'
+  def configure
+    credentials = {}
+    credentials['email'] = ask("Email:")
+    credentials['password'] = ask("Password:")
+    credentials['app_key'] = ask("App key:")
+    config_dir = File.dirname(CONFIG_PATH)
+    Dir.mkdir(config_dir) unless Dir.exist?(config_dir)
+    File.write(CONFIG_PATH, credentials.to_yaml)
+  end
+
+  desc 'list', 'List devices'
+  def list 
+    devices = client.appliance_list
+    head = [
+      { name: 'ID',     id: 'id'           },
+      { name: 'Name',   id: 'name'         },
+      { name: 'Type',   id: 'type'         },
+      { name: 'Online', id: 'onlineStatus' },
+      { name: 'Active', id: 'activeStatus' },
+    ]
+    print_table(devices, head)
+  end
+
+  desc 'get ID', 'Gets device\'s information'
+  def get id
+    command = MideaAirCondition::Command::RequestStatus.new
+    device = send_command(id, command)
+    puts "Device is turned #{(device.power_status ? 'on' : 'off')}."
+    puts "Target temperature: #{device.temperature} celsius"
+    puts "Indoor temperature: #{device.indoor_temperature} celsius"
+    puts "Outdoor temperature: #{device.outdoor_temperature} celsius"
+    puts "Mode: #{device.mode_human}"
+    puts "Fan speed: #{device.fan_speed}"
+    puts "TimerOn is #{(device.on_timer[:status] ? '' : 'not')} active."
+    puts "  at:  #{device.on_timer_human}" if device.on_timer[:status]
+    puts "TimerOff is #{(device.off_timer[:status] ? '' : 'not')} active."
+    puts "  at: #{device.off_timer_human}" if device.off_timer[:status]
+    puts "Eco mode is #{(device.eco ? 'on' : 'off')}."
+  end
+
+  desc 'set ID', 'Sets device\'s power, temperature and fan speed'
+  method_option :power, required: false, type: :string, aliases: "-p", desc: "Power"
+  method_option :target_temperature, required: false, type: :numeric, aliases: "-t", desc: "Target temperature"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special